.github/workflows/webui-periodic.yml

```diff
@@ -0,0 +1,111 @@
+# Run webui e2e tests periodically in Permian
+name: webui-periodic
+on:
+  #schedule:
+  #  - cron: 0 23 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  scenario:
+    name: Webui periodic
+    runs-on: [self-hosted, kstest]
+
+    timeout-minutes: 65
+    env:
+      # ocp-master-xxl: 32GB RAM / 4GB RAM per VM
+      TEST_JOBS: 8
+      # The timeout should be a few minutes less then the job's timeout-minutes
+      # so that we get partial results and logs in case of the timeout.
+      LAUNCHER_TIMEOUT_MINUTES: 60
+      # URL to the unpacked installation image
+      # TODO: use preview image
+      STATIC_REPO_URL: "http://10.43.136.2/users/rv/s2/webui-zveleba-f39"
+
+    steps:
+      # self-hosted runners don't do this automatically; also useful to keep stuff around for debugging
+      # need to run sudo as the launch script and the container create root/other user owned files
+      - name: Clean up previous run
+        run: |
+          sudo podman ps -q --all --filter='ancestor=kstest-runner' | xargs -tr sudo podman rm -f
+          sudo podman volume rm --all || true
+          sudo rm -rf *
+
+      # The test library is in anaconda repository
+      - name: Clone anaconda repository
+        uses: actions/checkout@v3
+        with:
+          ref: master
+          fetch-depth: 0
+          path: anaconda
+
+      # TODO: use main branch when the webui workflow is merged there
+      - name: Clone Permian repository
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: rhinstaller/permian
+          path: permian
+          ref: devel
+
+      - name: Clone tplib repository
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: rhinstaller/tplib
+          path: tplib
+
+      # Use static URL to unpacked image
+      # TODO: unpack the image as the part of the workflow
+      - name: Set the image to be tested
+        id: prepare_repo
+        run:
+          echo "repo_url=$STATIC_REPO_URL" >> $GITHUB_OUTPUT
+
+      - name: Create Permian settings file
```

Comment:
Did you consider using -o options instead of creating settings file every time? If you are changing only few settings it may look cleaner.

Reply:
Or, those settings are hard-coded anyway so maybe you should use static file stored in /anaconda/ui/webui/test/integration. Then you don't event have to specify it because permian looks for settings files in the library path.

Reply:
When creating GH workflow for kstests in Permian I used -o first, but then, based on a review I started to drop a file instead. It seems a bit cleaner / readable to me. And easier to modify.